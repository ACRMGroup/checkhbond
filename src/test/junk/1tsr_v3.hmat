residue ARG
donate	      17	      35	      30	     1
donate	      19	      36	      27	     1
donate	      20	      36	      27	     1
donate	      20	      37	      24	     1
donate	      21	      38	      32	     1
donate	      22	      34	      30	     1
donate	      22	      39	      35	     1
donate	      23	      35	      27	     1
donate	      23	      35	      29	     1
donate	      23	      36	      25	     1
donate	      24	      32	      40	     1
donate	      24	      35	      35	     1
donate	      24	      38	      24	     1
donate	      24	      38	      27	     1
donate	      24	      40	      30	     2
donate	      25	      36	      30	     1
donate	      25	      36	      33	     1
donate	      25	      37	      33	     1
donate	      25	      40	      29	     1
donate	      25	      41	      32	     1
donate	      25	      41	      33	     1
donate	      26	      37	      32	     1
donate	      26	      39	      36	     1
donate	      26	      40	      29	     1
donate	      26	      41	      27	     1
donate	      26	      41	      33	     1
donate	      27	      32	      36	     1
donate	      27	      35	      40	     1
donate	      27	      36	      26	     1
donate	      27	      40	      23	     1
donate	      27	      40	      32	     1
donate	      27	      42	      30	     1
donate	      27	      42	      32	     1
donate	      28	      38	      30	     2
donate	      28	      42	      37	     1
donate	      29	      38	      30	     1
donate	      29	      38	      34	     3
donate	      29	      42	      30	     1
donate	      30	      40	      27	     1
donate	      30	      40	      29	     1
donate	      30	      42	      31	     1
donate	      30	      42	      36	     1
donate	      30	      43	      36	     1
donate	      31	      38	      30	     1
donate	      31	      42	      27	     1
donate	      32	      30	      30	    17
donate	      32	      37	      30	     1
donate	      34	      39	      28	     1
partnertodonate	      22	      33	      39	     2
partnertodonate	      27	      37	      26	     2
partnertodonate	      30	      40	      29	     1
partnertodonate	      33	      33	      26	     1
partnertodonate	      34	      34	      28	     2
partnertodonate	      34	      39	      38	     1
partnertodonate	      37	      35	      30	     2
partnertodonate	      37	      40	      25	     1
accept	      27	      27	      29	     2
accept	      27	      28	      28	     1
accept	      27	      28	      29	     1
accept	      27	      29	      28	     2
accept	      28	      27	      28	     1
accept	      30	      28	      26	     1
accept	      30	      29	      26	     7
accept	      30	      30	      26	     2
partnertoaccept	      22	      27	      27	     1
partnertoaccept	      24	      21	      28	     1
residue THR
donate	      26	      32	      30	     1
donate	      27	      32	      29	     1
donate	      27	      32	      31	     1
donate	      29	      33	      32	     2
donate	      29	      34	      32	     1
donate	      30	      34	      28	     1
donate	      30	      34	      30	     1
donate	      30	      34	      31	     2
donate	      30	      34	      32	     4
donate	      32	      30	      30	    14
partnertodonate	      25	      36	      35	     1
partnertodonate	      27	      38	      31	     1
partnertodonate	      34	      36	      31	     2
partnertodonate	      34	      37	      30	     1
partnertodonate	      36	      35	      28	     1
accept	      26	      32	      30	     1
accept	      27	      27	      29	     1
accept	      27	      28	      28	     1
accept	      27	      29	      28	     2
accept	      27	      32	      29	     1
accept	      27	      32	      31	     1
accept	      28	      27	      28	     1
accept	      29	      33	      32	     2
accept	      29	      34	      32	     1
accept	      30	      28	      26	     1
accept	      30	      29	      26	     6
accept	      30	      30	      26	     2
accept	      30	      34	      28	     1
accept	      30	      34	      30	     1
accept	      30	      34	      31	     2
accept	      30	      34	      32	     4
partnertoaccept	      24	      30	      24	     1
residue ASN
donate	      25	      30	      31	     1
donate	      25	      35	      30	     1
donate	      25	      35	      31	     1
donate	      26	      34	      32	     1
donate	      30	      32	      34	     1
donate	      30	      34	      34	     1
donate	      30	      36	      32	     2
donate	      31	      34	      33	     1
donate	      32	      30	      30	     9
partnertodonate	      27	      30	      37	     1
partnertodonate	      35	      35	      28	     1
accept	      25	      30	      31	     1
accept	      25	      31	      30	     1
accept	      25	      34	      29	     1
accept	      27	      29	      27	     1
accept	      27	      29	      28	     1
accept	      27	      35	      33	     1
accept	      28	      28	      28	     1
accept	      28	      30	      26	     1
accept	      29	      30	      26	     1
accept	      29	      32	      34	     1
accept	      29	      33	      34	     1
accept	      30	      28	      26	     2
accept	      30	      29	      26	     2
accept	      30	      33	      34	     1
accept	      30	      36	      32	     1
accept	      30	      37	      30	     1
partnertoaccept	      24	      27	      27	     1
residue ASP
donate	      32	      30	      30	     8
accept	      24	      33	      30	     1
accept	      25	      31	      32	     1
accept	      25	      32	      31	     1
accept	      26	      31	      33	     1
accept	      27	      28	      28	     2
accept	      27	      28	      29	     1
accept	      27	      29	      28	     1
accept	      27	      34	      34	     1
accept	      27	      36	      28	     1
accept	      28	      33	      34	     1
accept	      29	      32	      34	     1
accept	      29	      36	      32	     1
accept	      30	      26	      28	     1
accept	      30	      28	      26	     1
accept	      30	      30	      26	     1
accept	      30	      33	      26	     1
accept	      30	      33	      34	     1
accept	      30	      34	      34	     1
accept	      30	      36	      32	     1
accept	      30	      37	      28	     1
accept	      31	      28	      26	     1
accept	      31	      33	      26	     1
accept	      31	      36	      31	     1
partnertoaccept	      21	      28	      30	     1
partnertoaccept	      27	      28	      30	     1
partnertoaccept	      31	      30	      22	     1
partnertoaccept	      34	      33	      22	     1
partnertoaccept	      36	      28	      27	     1
residue GLU
donate	      32	      30	      30	    11
accept	      23	      35	      29	     1
accept	      23	      35	      30	     1
accept	      23	      36	      30	     1
accept	      24	      34	      26	     1
accept	      24	      36	      27	     1
accept	      24	      37	      29	     1
accept	      25	      36	      29	     1
accept	      25	      36	      34	     1
accept	      26	      37	      30	     1
accept	      27	      29	      28	     1
accept	      27	      37	      30	     1
accept	      27	      37	      31	     1
accept	      27	      37	      35	     1
accept	      27	      38	      30	     2
accept	      27	      38	      31	     1
accept	      28	      27	      28	     2
accept	      28	      28	      27	     1
accept	      28	      36	      24	     1
accept	      28	      37	      26	     1
accept	      28	      38	      28	     1
accept	      28	      38	      30	     2
accept	      29	      38	      30	     1
accept	      30	      28	      26	     2
accept	      30	      29	      26	     2
accept	      30	      30	      26	     3
accept	      30	      38	      33	     1
partnertoaccept	      30	      34	      23	     1
residue GLN
donate	      23	      36	      30	     2
donate	      28	      38	      34	     2
donate	      30	      39	      29	     2
donate	      30	      39	      32	     2
donate	      30	      39	      33	     2
donate	      32	      30	      30	     7
donate	      32	      37	      26	     2
donate	      34	      37	      28	     2
accept	      27	      27	      28	     1
accept	      27	      28	      29	     1
accept	      27	      37	      32	     1
accept	      28	      30	      26	     1
accept	      29	      37	      30	     1
accept	      29	      38	      30	     1
accept	      30	      29	      26	     3
accept	      30	      30	      26	     1
accept	      32	      37	      30	     1
accept	      33	      32	      27	     1
accept	      33	      36	      30	     1
accept	      34	      35	      29	     1
partnertoaccept	      25	      22	      29	     1
partnertoaccept	      30	      30	      26	     1
partnertoaccept	      38	      30	      30	     1
residue LYS
donate	      21	      38	      30	     1
donate	      22	      34	      32	     1
donate	      22	      38	      29	     1
donate	      23	      40	      31	     1
donate	      31	      36	      37	     1
donate	      32	      30	      30	     5
accept	      27	      27	      28	     1
accept	      30	      28	      26	     1
accept	      30	      30	      26	     3
partnertoaccept	      31	      22	      27	     1
partnertoaccept	      32	      33	      26	     1
partnertoaccept	      34	      30	      24	     1
residue SER
donate	      26	      32	      30	     1
donate	      27	      32	      30	     1
donate	      27	      32	      32	     1
donate	      28	      33	      32	     1
donate	      29	      33	      28	     1
donate	      29	      33	      32	     1
donate	      30	      33	      32	     1
donate	      30	      34	      28	     1
donate	      30	      34	      29	     3
donate	      30	      34	      32	     3
donate	      31	      34	      29	     1
donate	      31	      34	      30	     1
donate	      32	      30	      30	    17
partnertodonate	      23	      30	      28	     1
partnertodonate	      24	      34	      34	     1
partnertodonate	      26	      37	      30	     1
partnertodonate	      35	      34	      30	     2
partnertodonate	      36	      32	      26	     2
partnertodonate	      36	      34	      30	     2
accept	      26	      32	      30	     1
accept	      27	      28	      28	     4
accept	      27	      28	      30	     1
accept	      27	      29	      28	     2
accept	      27	      32	      30	     1
accept	      27	      32	      32	     1
accept	      28	      27	      29	     1
accept	      28	      33	      32	     1
accept	      29	      30	      26	     1
accept	      29	      33	      28	     1
accept	      29	      33	      32	     1
accept	      30	      27	      27	     1
accept	      30	      28	      26	     3
accept	      30	      28	      27	     1
accept	      30	      29	      26	     3
accept	      30	      33	      32	     1
accept	      30	      34	      28	     1
accept	      30	      34	      29	     3
accept	      30	      34	      32	     3
accept	      31	      34	      29	     1
accept	      31	      34	      30	     1
partnertoaccept	      24	      30	      25	     1
partnertoaccept	      29	      33	      23	     1
partnertoaccept	      33	      37	      33	     1
residue TRP
donate	      31	      36	      35	     1
donate	      32	      30	      30	     1
accept	      30	      30	      26	     1
residue TYR
donate	      18	      31	      28	     1
donate	      18	      34	      30	     1
donate	      19	      33	      30	     1
donate	      30	      37	      39	     1
donate	      30	      38	      39	     2
donate	      32	      30	      30	     8
donate	      32	      37	      39	     1
donate	      32	      39	      38	     1
accept	      18	      31	      28	     1
accept	      18	      34	      30	     1
accept	      19	      33	      30	     1
accept	      27	      28	      28	     1
accept	      28	      30	      26	     1
accept	      30	      29	      26	     4
accept	      30	      30	      26	     2
accept	      30	      37	      39	     1
accept	      30	      38	      39	     2
accept	      32	      37	      39	     1
accept	      32	      39	      38	     1
partnertoaccept	      22	      27	      31	     1
partnertoaccept	      26	      26	      34	     1
partnertoaccept	      30	      37	      45	     1
residue HIS
donate	      25	      32	      31	     1
donate	      25	      33	      29	     1
donate	      28	      35	      34	     1
donate	      30	      35	      33	     1
donate	      30	      36	      32	     2
donate	      31	      36	      32	     1
donate	      32	      30	      30	     7
partnertodonate	      36	      33	      30	     1
accept	      25	      32	      31	     1
accept	      25	      33	      29	     1
accept	      27	      28	      29	     1
accept	      27	      29	      28	     1
accept	      28	      27	      29	     1
accept	      28	      35	      34	     1
accept	      29	      30	      26	     1
accept	      30	      29	      26	     2
accept	      30	      30	      26	     1
accept	      30	      35	      33	     1
accept	      30	      36	      32	     2
accept	      31	      36	      32	     1
residue ALA
donate	      32	      30	      30	     7
partnertodonate	      35	      34	      30	     1
accept	      27	      28	      28	     2
accept	      30	      29	      26	     4
accept	      31	      30	      26	     1
partnertoaccept	      33	      31	      23	     1
residue CYS
donate	      32	      30	      30	    10
accept	      27	      29	      27	     1
accept	      27	      30	      27	     1
accept	      29	      30	      26	     2
accept	      30	      28	      26	     3
accept	      30	      29	      26	     3
partnertoaccept	      29	      31	      21	     1
residue PHE
donate	      32	      30	      30	     5
accept	      27	      28	      29	     1
accept	      30	      29	      26	     3
accept	      30	      30	      26	     1
residue GLY
residue ILE
donate	      32	      30	      30	     6
accept	      30	      28	      26	     1
accept	      30	      29	      26	     2
accept	      30	      30	      26	     3
residue LEU
donate	      32	      30	      30	    14
partnertodonate	      33	      32	      25	     1
accept	      27	      27	      28	     1
accept	      27	      28	      28	     2
accept	      27	      29	      28	     1
accept	      28	      27	      28	     1
accept	      30	      27	      27	     1
accept	      30	      28	      26	     1
accept	      30	      29	      26	     4
accept	      30	      30	      26	     3
residue MET
donate	      32	      30	      30	     6
accept	      26	      28	      29	     1
accept	      27	      28	      28	     2
accept	      30	      29	      26	     2
accept	      30	      30	      26	     1
partnertoaccept	      22	      29	      26	     1
partnertoaccept	      22	      32	      30	     1
partnertoaccept	      24	      33	      27	     1
partnertoaccept	      27	      22	      29	     1
residue PRO
donate	      32	      30	      30	    14
partnertodonate	      34	      30	      24	     1
accept	      27	      27	      29	     2
accept	      27	      28	      28	     1
accept	      27	      28	      29	     1
accept	      30	      28	      26	     1
accept	      30	      29	      26	     8
accept	      30	      30	      26	     1
partnertoaccept	      24	      27	      34	     1
partnertoaccept	      34	      30	      23	     1
residue VAL
donate	      32	      30	      30	    15
partnertodonate	      35	      32	      26	     1
partnertodonate	      37	      32	      26	     1
accept	      30	      28	      26	     3
accept	      30	      29	      26	     7
accept	      30	      30	      26	     5
partnertoaccept	      30	      33	      24	     1
partnertoaccept	      35	      32	      26	     1
