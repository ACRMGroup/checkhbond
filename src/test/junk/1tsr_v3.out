INFO: Processing residue type ARG
INFO: Processing file /tmp//1tsr
ARG    110 NE   : GLY    108 O    HE   C   
ARG    110 NH2  : GLY    108 O     ??? C   
ARG    175 NH1  : PRO    191 O     ??? C   
ARG    175 NE   : MET    237 O    HE   C   
ARG    175 NH2  : MET    237 O     ??? C   
ARG    202 N    : ASN    200 OD1   ??? CG  
ARG    249 NH2  : GLY    245 O     ??? C   
ARG    249 N    : MET    246 O     ??? C   
ARG    249 NE   : MET    246 O    HE   C   
ARG    267 NE   : LYS    101 O    HE   C   
ARG    267 NH2  : LYS    101 O     ??? C   
THR    284 N    : ARG    280 O     ??? C   
THR    284 OG1  : ARG    280 O     ??? C   
ARG    283 NH2  : THR    118 O     ??? C   
INFO: Processing residue type THR
INFO: Processing file /tmp//1tsr
ARG    283 NH2  : THR    118 O     ??? C   
THR    125 OG1  : GLY    117 O     ??? C   
THR    155 OG1  : PRO    152 O     ??? C   
THR    170 OG1  : GLN    167 O     ??? CA  
THR    230 OG1  : GLU    221 O     ??? C   
THR    284 N    : ARG    280 O     ??? C   
THR    284 OG1  : ARG    280 O     ??? C   
INFO: Processing residue type ASN
INFO: Processing file /tmp//1tsr
ARG    202 N    : ASN    200 OD1   ??? CG  
ASN    235 ND2  : LYS    139 O     ??? C   
ASN    263 N    : SER    261 OG    ??? CB  
INFO: Processing residue type ASP
INFO: Processing file /tmp//1tsr
SER    183 OG   : ASP    184 O     ??? C   
LEU    188 N    : ASP    186 OD1   ??? CG  
ALA    189 N    : ASP    186 OD1   ??? CG  
SER    260 N    : ASP    259 OD1   ??? CG  
SER    260 OG   : ASP    259 OD1   ??? CG  
INFO: Processing residue type GLU
INFO: Processing file /tmp//1tsr
THR    230 OG1  : GLU    221 O     ??? C   
INFO: Processing residue type GLN
INFO: Processing file /tmp//1tsr
HIS    168 N    : GLN    167 OE1   ??? CD  
THR    170 OG1  : GLN    167 O     ??? CA  
PRO    191 N    : GLN    192 OE1   ??? CD  
INFO: Processing residue type LYS
INFO: Processing file /tmp//1tsr
ARG    267 NE   : LYS    101 O    HE   C   
ARG    267 NH2  : LYS    101 O     ??? C   
ASN    235 ND2  : LYS    139 O     ??? C   
INFO: Processing residue type SER
INFO: Processing file /tmp//1tsr
SER    116 OG   : CYS    124 O     ??? C   
SER    149 N    : TYR    107 O     ??? C   
SER    149 OG   : TYR    107 O     ??? C   
SER    183 OG   : ASP    184 O     ??? C   
GLY    187 N    : SER    185 OG    ??? CB  
SER    185 OG   : ALA    189 O     ??? C   
SER    240 N    : VAL    274 O     ??? C   
SER    240 OG   : VAL    274 O     ??? C   
VAL    274 N    : SER    240 OG    ??? CB  
SER    260 N    : ASP    259 OD1   ??? CG  
SER    260 OG   : ASP    259 OD1   ??? CG  
ASN    263 N    : SER    261 OG    ??? CB  
INFO: Processing residue type TRP
INFO: Processing file /tmp//1tsr
INFO: Processing residue type TYR
INFO: Processing file /tmp//1tsr
SER    149 N    : TYR    107 O     ??? C   
SER    149 OG   : TYR    107 O     ??? C   
VAL    197 N    : TYR    205 OH    ??? CZ  
INFO: Processing residue type HIS
INFO: Processing file /tmp//1tsr
HIS    168 N    : GLN    167 OE1   ??? CD  
INFO: Processing residue type ALA
INFO: Processing file /tmp//1tsr
SER    185 OG   : ALA    189 O     ??? C   
ALA    189 N    : ASP    186 OD1   ??? CG  
INFO: Processing residue type CYS
INFO: Processing file /tmp//1tsr
SER    116 OG   : CYS    124 O     ??? C   
INFO: Processing residue type PHE
INFO: Processing file /tmp//1tsr
INFO: Processing residue type GLY
INFO: Processing file /tmp//1tsr
INFO: Processing residue type ILE
INFO: Processing file /tmp//1tsr
INFO: Processing residue type LEU
INFO: Processing file /tmp//1tsr
LEU    188 N    : ASP    186 OD1   ??? CG  
INFO: Processing residue type MET
INFO: Processing file /tmp//1tsr
ARG    175 NE   : MET    237 O    HE   C   
ARG    175 NH2  : MET    237 O     ??? C   
ARG    249 N    : MET    246 O     ??? C   
ARG    249 NE   : MET    246 O    HE   C   
INFO: Processing residue type PRO
INFO: Processing file /tmp//1tsr
THR    155 OG1  : PRO    152 O     ??? C   
ARG    175 NH1  : PRO    191 O     ??? C   
PRO    191 N    : GLN    192 OE1   ??? CD  
INFO: Processing residue type VAL
INFO: Processing file /tmp//1tsr
VAL    197 N    : TYR    205 OH    ??? CZ  
VAL    274 N    : SER    240 OG    ??? CB  
SER    240 N    : VAL    274 O     ??? C   
SER    240 OG   : VAL    274 O     ??? C   
